mitodl/ocw-studio
# convert resource_file to resource shortcodes

#### Pre-Flight checklist


#### What are the relevant tickets? #1012 

#### What's this PR do?
This refactors `markdown_cleanup_baseurl` management command to be `markdown_cleanup` which should now be invoked with one of multiple rules. Right now, one of:
```
# Either "baseurl" or "resource_file_to_resource"
# plus the old
docker-compose run --rm web python manage.py markdown_cleanup baseurl -o out.csv
docker-compose run --rm web python manage.py markdown_cleanup resource_file_to_resource -out.csv
```
or with `--commit` to commit the result.

There are no changes to the `baseurl` command besides refactoring. I have tested it locally with a baseurl-intact database.

#### How should this be manually tested?
Run the commands above and verify resource_file links are converted as expected.

The "Instructor Insights" page of `8-591j-systems-biology-fall-2014` is a good one to check in studio. See screenshots below.

#### Screenshots (if appropriate)

**Before**
<img width="600" alt="Screen Shot 2022-02-16 at 4 10 02 PM" src="https://user-images.githubusercontent.com/9010790/154357821-9a702595-c21f-4fea-aabf-2b63276f9515.png">

**After**
<img width="600" alt="Screen Shot 2022-02-16 at 4 09 22 PM" src="https://user-images.githubusercontent.com/9010790/154357839-bf3d8338-f088-4cc0-b080-fa8e2a244df2.png">
